dom/events.ts

```diff
@@ -207,6 +214,66 @@ export function trigger (element : null|EventTarget, type : string, data : any =
 }
 
 
+/**
+ * Adds media query listener
+ */
+export function mediaQuery (
+    query: string,
+    callback: (mediaMatches: boolean, targetWindow: Window|null) => {},
+    executeImmediately: boolean = false,
+    targetWindow: Window = window
```

Comment:
We can remove this one, as we will likely never change it.

Reply:
On some internal projects we already needed to call machMedia on a different `window` object.

Reply:
I've just searched our entire GitHub org for `matchMedia` and scrolled through all 6 pages. All of them were calling it on the `window` object.

Reply:
yeah, in an one-on-one talk we decided not to include it.